app/Http/Controllers/OfferController.php

```
@@ -344,108 +153,39 @@ public function uploadTempImages(Request $request)
-            message: $message,
-            user: $user,
-            offer: $offer,
-            buyer: User::findOrFail((int) $offer->buyer_id),
-            actionType: 6
-        );
-
-        broadcast(new \App\Events\MessageSent($message));
+        $this->offerTransactionService->receiveOffer($offer);
     }
 
     public function cancelOffer(Request $request, Offer $offer)
```

PHPMD (2.15.0)

Avoid unused parameters such as '$request'. (Unused Code Rules)

(UnusedFormalParameter)